.gitignore

```diff
@@ -1,23 +1,37 @@
-# Compiled class file
-*.class
```

Comment:
i miss IntelliJ-configuration files here. You should also keep "Class"

Reply:
ohh never mind! I was confused because of the still unfamiliar view of code changes in github